src/ExperianModels/AdditionalData.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace CustomD\LaravelExperian\ExperianModels;
+
+class AdditionalData extends Base
```

Comment:
With these would be good to add the type declarations, this allows for our editors to autocomplete when typing code.

```suggestion
/**
 * \CustomD\LaravelExperian\ExperianModels\AdditionalData
 *
 * @property string $DrivingLicenceNumber 
 * @property string $VehicleRegistration 
 * @property string $PlaceOfBirth 
 * @property string $MothersMaidenName 
 * @property string $BirthSurname
 */
class AdditionalData extends Base
```